pkg/util/restoptions/configgetter.go

```diff
@@ -26,6 +26,10 @@ import (
 // UseConfiguredCacheSize indicates that the configured cache size should be used
 const UseConfiguredCacheSize = -1
 
+// DefaultCacheSize is the cache size to use for etcdHelper
+// TODO: Make this configurable in future
+const DefaultCacheSize = 100
```

Comment:
That option is already overrideable via an apiServerArgs `--deserialization-cache-size` setting... we should use that instead of setting our own separate non-overrideable default